trymito.io/pages/teams.tsx

```diff
@@ -0,0 +1,232 @@
+                </p>
+                <div className={homeStyles.cta_button_and_video_spacer}>
+                    <CTAButtons variant='contact' />
+                </div>
+            </section>
+            <section>
+                <div className={pageStyles.subsection}>
+                    <div className={homeStyles.functionality_text}>
+                        <p>
+                            <b className='text-primary'>Spreadsheet-first teams </b> use Mito to transition to Python while preserving 
+                            the spreadsheet interface that we all know and love. 
+                        </p>
+                        <p> 
+                            <b className='text-primary'>Python-first teams </b> use Mito to write code faster without spending time searching Google or Stack Overflow.
+                        </p>
+                        <a href="mailto:founders@sagacollab.com" className={pageStyles.link_with_p_tag_margins}>
+                            Contact the Mito team →
+                        </a>
+                    </div>
+                    <div className={homeStyles.functionality_media}>
+                        <Image src={'/explore.png'} alt='Explore your data with Mito' width={500} height={250} layout='responsive'/>
+                    </div>
+                </div>
+            </section>
+            <section>
+                <div className={pageStyles.subsection + ' center'}>
+                    <h1>
+                        Deploy Mito in Jupyter in minutes
+                    </h1>
+                </div>
+                <div className={pageStyles.subsection}>
+                    <div className={securityStyles.security_bullet_container}>
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={JupyterIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Jupyter <br/> Compatible
+                        </h1>
+                        <p>
+                            Mito is compatible with Jupyter Hub, Jupyter Lab, and Jupyter Notebooks, so you don’t need to manage any new infrastructure.  
+                        </p>
+                    </div>
+                    <div className={securityStyles.security_bullet_container + ' ' + pageStyles.subsection_second_element_mobile_spacing}>
+                        <div className={securityStyles.icon}>
+                            <Image className={securityStyles.icon} src={PrivateIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Completely <br/> Private
+                        </h1>
+                        <p>
+                            With the Mito Pro and Enterprise plans, not a single message is sent back to Mito. Mito is designed from the ground up with security at its core. 
```

Comment:
I think it's fine to keep this focused on enterprise. Also, "a message" sounds a bit strange.

- The Mito Enterprise plan is totally local - no data leaves your computers, ever. Mito is designed for your security, not for our profit.
- The Mito Enterprise plan is totally local, no data leaves your computer, ever. Mito is designed for your security first.


- The Mito Enterprise plan is totally local. Mito is designed from the ground up with security at its core.